src/main/resources/swagger/api-docs.yaml

```diff
@@ -155,10 +155,17 @@ paths:
       parameters:
         - name: email
           in: path
-          description: Email address of user to check
+          description: Email address (or user id) of user to check
```

Comment:
It was in the ticket to support both user email and id. I realize the api calls this field email. Changing the api to be clear that both email and id are supported is just so much more work.